### PR TITLE
Add details on how to add scopes for a resource

### PR DESCRIPTION
**Pull request by npamudika**
Fixes related to https://github.com/wso2/product-apim-tooling/issues/258

### CODE REVIEW

File: en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md [hunk 1]
```diff
@@ -86,6 +86,57 @@ WSO2 API Controller, **apictl** allows to create and deploy APIs without using W
...
+                securitySchemes: 
+                    default: 
+                    type: "oauth2"
+                    flows: 
+                        implicit: 
+                        authorizationUrl: "https://test.com"
+                        scopes: 
+                            products:read: ""
+                        x-scopes-bindings: 
+                            products:read: "admin"
```
Comment (malinthaprasan) [inline en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md:134]: Should this be indented?

Comment (malinthaprasan) [reply en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md:134]:
```suggestion
                        type: "oauth2"
                        flows: 
                            implicit: 
                            authorizationUrl: "https://test.com"
                            scopes: 
                                products:read: ""
                            x-scopes-bindings: 
                                products:read: "admin"
```

File: en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md [hunk 2]
```diff
@@ -86,6 +86,57 @@ WSO2 API Controller, **apictl** allows to create and deploy APIs without using W
...
+                ```
+
+            First you need to define the scope name (products:read) under `security > default` section inside the required resource and then define the role binding under the `securitySchemes` section.
+
+            Make sure to set the security name as **`default`** when defining the scopes.
```
Comment (malinthaprasan) [inline en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md:139]: Let's remove this line if we fixed https://github.com/wso2/product-apim/issues/7648

File: en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md (hunk 1 above)
Comment (npamudika) [reply en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md:134]: yes. will correct it.

File: en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md (hunk 2 above)
Comment (npamudika) [reply en/docs/learn/api-controller/importing-apis-via-dev-first-approach.md:139]: will be tracked via https://github.com/wso2/product-apim-tooling/issues/271

**Pull request merged**